src/Symfony/Component/RateLimiter/RateLimiterFactory.php

```diff
@@ -68,19 +68,21 @@ public function create(?string $key = null): LimiterInterface
     protected static function configureOptions(OptionsResolver $options): void
     {
         $intervalNormalizer = static function (Options $options, string $interval): \DateInterval {
-            try {
-                // Create DateTimeImmutable from unix timesatmp, so the default timezone is ignored and we don't need to
-                // deal with quirks happening when modifying dates using a timezone with DST.
-                $now = \DateTimeImmutable::createFromFormat('U', time());
+            // Create DateTimeImmutable from unix timesatmp, so the default timezone is ignored and we don't need to
+            // deal with quirks happening when modifying dates using a timezone with DST.
+            $now = \DateTimeImmutable::createFromFormat('U', time());
 
-                return $now->diff($now->modify('+'.$interval));
```

Comment:
`modify()` returns `false` on failure, thus we must not pass it to `diff()` before checking the type